.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,32 @@
+name: Pull request to main
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  ci:
+    name: PR build and CI
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Is an infinity pull still needed?

Reply:
No, shouldn't be needed... will remove it. Thanks!